src/sentry/static/sentry/app/views/settings/components/settingsProjectItem.jsx

```diff
@@ -0,0 +1,93 @@
+import styled from 'react-emotion';
+import React from 'react';
+
+import createReactClass from 'create-react-class';
+import theme from '../../../utils/theme';
+
+import {update} from '../../../actionCreators/projects';
+import ApiMixin from '../../../mixins/apiMixin';
+import Tooltip from '../../../components/tooltip';
+import Link from '../../../components/link';
+import ProjectLabel from '../../../components/projectLabel';
+import SentryTypes from '../../../proptypes';
+
+const Header = styled('h5')`
```

Comment:
What exactly are you getting out of this element? Judging by the screenshots it looks like perhaps a little more text size and boldness? If so, I would just apply the one or two styles you want to a div and skip the dependency on the`h5`. Using html elements only for their styles is usually not a good idea.

Reply:
this is a good point, but i'm, not sure what text size and boldness I actually want so i just wanted to piggy pack off of some standard hierarchy. short of just reading what h5 is applying and copying it over, do you have a suggestion for getting these styles?

Reply:
Lemme look into it. I'll get back shortly!

Reply:
thank you :pra